src/Access/AccessControl.cpp

```diff
@@ -79,7 +79,7 @@ class AccessControl::ContextAccessCache
             /// No user, probably the user has been dropped while it was in the cache.
             cache.remove(params);
         }
-        auto res = std::shared_ptr<ContextAccess>(new ContextAccess(access_control, params));
+        auto res = ContextAccess::make(access_control, params);
```

Comment:
A plain and simple `auto res = std::make_shared<ContextAccess>(access_control, params);` would have done the job but I am also fine with `::make()`.